src/posh-git.psd1

```diff
@@ -65,16 +65,16 @@ PrivateData = @{
         Tags = @('git', 'prompt', 'tab', 'tab-completion', 'tab-expansion', 'tabexpansion')
 
         # A URL to the license for this module.
-        LicenseUri = 'https://github.com/dahlbyk/posh-git/blob/v1.0.0-beta1/LICENSE.txt'
+        LicenseUri = 'https://github.com/dahlbyk/posh-git/blob/develop/LICENSE.txt'
 
         # A URL to the main website for this project.
         ProjectUri = 'https://github.com/dahlbyk/posh-git'
 
         # ReleaseNotes of this module
-        ReleaseNotes = 'https://github.com/dahlbyk/posh-git/blob/v1.0.0-beta1/CHANGELOG.md'
+        ReleaseNotes = 'https://github.com/dahlbyk/posh-git/blob/develop/CHANGELOG.md'
 
         # OVERRIDE THIS FIELD FOR PUBLISHED RELEASES - LEAVE AT 'alpha' FOR CLONED/LOCAL REPO USAGE
-        Prerelease = 'beta1'
+        Prerelease = 'beta1x'
```

Comment:
This is for folks who have git cloned and then import right?  I assume this will cause repo bits to have precedence over the installed beta1.  Is that the idea with `beta1x`?

Reply:
Yes. `beta1 < beta1x < beta2`